clients/admin-ui/src/features/config-wizard/PrivacyDeclarationForm.tsx

```diff
@@ -0,0 +1,387 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  FormLabel,
+  Heading,
+  Stack,
+  Text,
+  Tooltip,
+  useToast,
+} from "@fidesui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query/fetchBaseQuery";
+import { Form, Formik } from "formik";
+import type { NextPage } from "next";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { AddIcon, QuestionIcon } from "~/features/common/Icon";
```

Comment:
nit: looks like this isnt new but `Icon` should be changed to lowercase from what I can tell. `~/features/common/icon`

Reply:
This is referencing the Icon folder, which is capitalized in the codebase.